nix/cells/cloud/actions.nix

```diff
@@ -21,4 +19,26 @@
       ]
     '';
   };
+
+  "plutus/publish-unraveling-recursion-paper" = {
+    task = "publish-unraveling-recursion-paper";
+    io = ''
+      let push = {
+        #lib.io.github_push
+        #input: "${cell.library.actions.paper.input}"
+        #repo: "input-output-hk/plutus"
+        inputs: _final_inputs
+      }
+
+      _final_inputs: inputs
+      inputs: {
+        push.inputs
+
+        "CI passed": match: {
```

Comment:
This means this won't run unless the CI job passed, I assume? What links it to that? I don't see anything that sets "ok" in the CI action? Should the CI action publish some specific fact?

Reply:
`#lib.io.github_push` publishes

```
{
  "ok": true,
  "revision": …
}
```

[by default](https://github.com/input-output-hk/tullia/blob/86c97fe01e69070ae7f68e745f9bf6335f483711/lib/github.cue#L116-L122).

Reply:
I should learn CUE I guess :thinking:

Reply:
Does this not cause a cyclic dependency on action? Thats what happened to me when I tried to do something similar with the comment action.